README.md

```diff
@@ -88,10 +88,17 @@ Below are the possible installation options.
    $ source .env/bin/activate  # Linux
    ```
 
-3. Install dependencies:
+3. We need to downgrade `setuptools` because the `mailer` package won't install in
+   the latest `setuptools` versions (it uses the removed `use_2to3` argument to `setup`):

```

Comment:
This is not related to this PR, it was happening before (we don't see in the tests because the docker image probably has a setuptools version older than 58).